02-wrangling.Rmd

```diff
@@ -146,20 +202,16 @@ lang_wide
-
-To accomplish this data transformation, we will use the `tidyverse` function `pivot_longer`. To use `pivot_longer` we need to specify the:
 
-1. `data`: the data set
+What is wrong with our untidy format above? From a data analysis perspective, this format is not ideal because, in this format, the outcome of the variable *region* (Toronto, Montréal, Vancouver, Calgary and Edmonton) is stored as column names. Thus it is not easily accessible for the data analysis functions we will want to apply to our data set. Additionally, the values of the *mother tongue* variable are spread across multiple columns, which will prevent us from doing any desired visualization or statistical tasks until we combine them into one column. For instance, suppose we want to know the languages with the highest number of Canadians reporting it as their mother tongue among all five regions. This question would be tough to answer with the data in its current format. It would be much easier to answer if we tidy our data first. 
 
-2. `cols` : the names of the columns that we want to combine
+To accomplish this data transformation, we will use the `tidyverse` function `pivot_longer`. Figure \@ref(fig:img-pivot-longer) details what we need to specify to use `pivot_longer`.
 
-3. `names_to`: the name of a new column that will be created, whose values will come from the *names of the columns* that we want to combine 
-
-4. `values_to`: the name of a new column that will be created, whose values will come from the *values of the columns* we want to combine 
-
-For the above example, we use `pivot_longer` to combine the Toronto, Montreal, Vancouver, Calgary and Edmonton columns into a single column called `region`, and create a column called `mother_tongue` that contains the count of how many Canadians report each language as their mother tongue for each metropolitan area. We use a colon `:` between Toronto and Edmonton tells R to select all the columns in between Toronto and Edmonton: 
+```{r img-pivot-longer, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Syntax for the pivot_longer function", out.width="1100", fig.retina = 2}
+knitr::include_graphics("img/pivot_longer.jpeg")
```

Comment:
This figure is moving well in the right direction! Nice work!!! I think we can improve it by changing it slightly so we don't have the lines draw over the the tables, something like this:

![PXL_20210606_061454499](https://user-images.githubusercontent.com/6865016/120914619-4610f100-c654-11eb-9cee-4b020aec9b82.jpg)